tracer/test/test-applications/integrations/Samples.DataStreams.RabbitMQ/Program.cs

```diff
@@ -21,53 +31,159 @@ public static async Task Main(string[] args)
             using var connection = factory.CreateConnection();
             using var model = connection.CreateModel();
         
+            // produce/consume operation (default exchange)
+            PublishMessageToDefaultExchange(model);
+            GetMessage(model, DefaultQueue);
+
             // produce/consume operation (direct exchange)
-            PublishMessageToQueue(model, Queue, Message);
-            var msg = GetMessage(model, Queue);
+            List<string> directQueues = PublishMessageToDirectExchange(model);
+            foreach (var queue in directQueues) {
+                GetMessage(model, queue);
+            }
```

Comment:
tiny nit: most IDEs will default to the following, so changing now before something automatically tidies it up 🙂 

```suggestion
            foreach (var queue in directQueues) 
            {
                GetMessage(model, queue);
            }
```